client/profile-wizard/style.scss

```diff
@@ -297,7 +297,6 @@
 
 		.components-base-control__label {
 			white-space: nowrap;
-			overflow: hidden;
```

Comment:
Seems the reason this was added was to prevent styling issues on mobile.

Related PR - https://github.com/woocommerce/woocommerce-admin/pull/3156

Reply:
I was going to suggest using a media query to avoid applying the change on mobile, but if the overflow issue happens on mobile that's another regression. It would be nice to be able to reproduce this in the normal way.

Reply:
Adding `padding: 10px 0` in the dev tools seems to fix things... not sure if there are other side-effects though!

Reply:
We need to add this back...

<img width="427" alt="Screen Shot 2022-01-28 at 16 12 34" src="https://user-images.githubusercontent.com/2098816/151622683-eadb1c44-348d-4713-8fea-328826df18ec.png">